features/mobile/likes.feature

```diff
@@ -0,0 +1,22 @@
+@javascript @mobile
+Feature: Liking posts
+  In order to show my appreciation
+  As a friendly person
+  I want to like their posts
+
+  Background:
+    Given following users exist:
+      | username    | email             |
+      | Bob Jones   | bob@bob.bob       |
+      | Alice Smith | alice@alice.alice |
+    And a user with email "bob@bob.bob" is connected with "alice@alice.alice"
+    And "bob@bob.bob" has a public post with text "I like unicorns"
+    And I sign in as "alice@alice.alice"
```

Comment:
The other tests have `And I sign in as "bob@bob.bob" **on the mobile website**`. I don't know if that change anything, I ran the test locally and it was correctly using the mobile version.  I wanted to point that just in case.